core/src/com/unciv/models/ruleset/Era.kt

```diff
@@ -40,6 +42,10 @@ class Era : INamed {
 
     fun getHexColor() = "#" + getColor().toString().substring(0, 6)
 
+    fun hasUnique(uniqueTemplate: String) = uniqueObjects.any { it.placeholderText == uniqueTemplate }
```

Comment:
Maybe we should have an interface for Inamed with uniques, this is getting repetitive